app/views/course/assessment/answer/programming/_annotation_topic.html.slim

```diff
@@ -1,2 +1,3 @@
-= render partial: 'course/discussion/topic', object: annotation_topic,
-         locals: { post_partial: 'course/assessment/answer/programming/annotation_post' }
+- unless annotation_topic.posts.empty?
```

Comment:
I'm not sure if the existing code causes an N+1 query, but it looks like it might. I didn't run this... can you check the logs?